spec/factories/books.rb

```diff
@@ -32,6 +32,6 @@
     author_name { 'レオ＝レオニ' }
     author_image { File.new("#{Rails.root}/spec/factories/images/test2.png") }
     price { 600 }
-    isbn { 4-7690-2017-1 }
+    isbn { 4 - 7690 - 2017 - 1 }
```

Comment:
これ数字じゃなくて、文字列にしたいんですかね？　であればこうです。

```suggestion
    isbn { "4 - 7690 - 2017 - 1" }
```

今だと数字の引き算の結果になってないですか？ → `-9704`
カラムがintegerならstringに変える必要があります。